gitpod-io/gitpod
# [code] Update stable to 1.66.0

## Description

Updates stable VS Code Web to 1.66.0

## Related Issue(s)
This is a part of #9020 

## How to test

- Make sure `VS Code` is selected in the preferences as the `Browser Editor` (should be by default)
- Start a workspace
- Use the `Help -> About` about dialog to check that version is 1.66.0.

## Release Notes

```release-note
NONE
```
